### ISSUE TITLE
complex float input and output

### ISSUE THREAD

**Issue by mossmann** [mossmann/hackrf#71]
hackrf_transfer should have an option to store samples in complex float format (GNU Radio's complex data type) for compatibility.  A switch (-c for complex float?) could enable this mode instead of the default unsigned 8 bit format for both -r and -t operations.

**Comment by mossmann** [comment]
I'm closing this because doing the conversion in GNU Radio or elsewhere makes more sense.  For examples of ways to convert on the command line, see the homework on: [http://greatscottgadgets.com/sdr/11/](http://greatscottgadgets.com/sdr/11/)

**Comment by krysiek** [comment]
How can I import or convert SDRSharp captured data on RTL2832 usb dongle to hackrf_transfer? I would like to resend these old data which I have captured and currently I cannot record and resend it as its easy in hackrf_transfer with -r -t switch.
SDRSharp save 8 bit unsigned wav files whereas hackrf_transfer need 8bit signed without wav header, if I am correct. I have gnu radio companion but I tried to add -127 to the samples but I am mixed alittle bit now.
